src/services/LibraryService.js

```diff
@@ -33,3 +33,14 @@ export async function remove(id) {
   return api.delete(`/api/classes/Library/${id}`, { headers: getDefaultHeaders() })
     .catch(manageError);
 }
+
+/**
+ * Create library.
+ * @param {object} library - Library to create.
```

Comment:
object containing required/optional parameter to create a library, not a library.

Reply:
done